Makefile

```diff
@@ -56,4 +60,7 @@ promu:
 		GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
 		$(GO) get -u github.com/prometheus/promu
 
-.PHONY: all style format build test vet tarball docker promu
\ No newline at end of file
+release:
+	go build -ldflags="$(GO_BUILD_LDFLAGS)" -o $(PMM_RELEASE_PATH)/azure_exporter
```

Comment:
shouldn't it be `azure_metrics_exporter`?

Reply:
Should be azure_exporter like in previous FBs